test/rules/src/https_everywhere_checker/rules.py

```diff
@@ -177,6 +183,15 @@ def _determineTestApplication(self):
                  off that rule or exclusion.  Return any coverage problems."""
         if not self.determine_test_application_run:
             self.test_application_problems = []
+
+            # check for duplicated test urls
+            if len(self.tests) != len(set(self.tests)):
+                uniqueTests = set(self.tests)
```

Comment:
We're not using this anywhere.  We should change the below line to be `for test in set(self.tests):`